django/views/debug.py

```diff
@@ -259,6 +259,11 @@ def __init__(self, request, exc_type, exc_value, tb, is_email=False):
         self.template_info = getattr(self.exc_value, 'template_debug', None)
         self.template_does_not_exist = False
         self.postmortem = None
+        # Avoid using __file__ at module level to be kind to frozen Python environments
```

Comment:
Could this comment be a docstring on the helper function? 

I'd probably rephrase slightly.

Reply:
Is 93124234f2febfc48e92a910243f42f4ebf5d72a responsive to this request?